site/book/03-packages/03-editing-a-package.md

```diff
@@ -0,0 +1,49 @@
+kpt does not maintain any state on your local machine outside of the directory where you fetched the
+package. Making changes to the package is accomplished by manipulating the local filesystem. At the
+lowest-level, _editing_ a package is simply a process that either:
+
+- Changes the resources within that package. Examples:
+  - Authoring new a Deployment resource
+  - Customizing an existing Deployment resource
+  - Modifying the Kptfile or other meta resources
+- Changes the package hierarchy, also called _package composition_. Examples:
+  - Adding a subpackage.
+  - Create a new dependent subpackage.
+
+At the end of the day, editing a package will result in a Git commit that fully specifies
+the package. This process can be manual or automated depending on your use case.
+
+We will cover package composition later in this chapter. For now, let's focus on editing resources
+_within_ a package.
+
+## Manual edits
+
+As mentioned earlier, you can manually edit or _author_ KRM resources using your favorite editor.
```

Comment:
these terms are highlighted but I am not sure why.  Are these concepts, that are defined else where?  Do we want the user to somehow focus on these?  It's not a big deal but just trying to map this pattern to other documents that I have seen.